buildenv/jenkins/aqaTestPipeline.groovy

```diff
@@ -150,16 +150,23 @@ def generateJobs(jobJdkVersion, jobTestFlag, jobPlatforms, jobTargets) {
             def VENDOR_TEST_BRANCHES = ''
             def VENDOR_TEST_DIRS = ''
             int rerunIterations = params.RERUN_ITERATIONS ? params.RERUN_ITERATIONS.toInteger() : 0
+            def buildList = params.BUILD_LIST ?: ""
             if (params.VARIANT == "openj9") {
                 // default rerunIterations is 3 for openj9
                 rerunIterations = params.RERUN_ITERATIONS ? params.RERUN_ITERATIONS.toInteger() : 3
                 if (TARGET.contains('external')) {
                     PARALLEL = "None"
                     rerunIterations = 0
                 } else if (TARGET.contains('functional')) {
-                    VENDOR_TEST_REPOS = 'git@github.ibm.com:runtimes/test.git'
-                    VENDOR_TEST_BRANCHES = params.ADOPTOPENJDK_BRANCH ?: 'master'
-                    VENDOR_TEST_DIRS = 'functional'
+                    if (jobTestFlag.contains("FIPS")) {
+                        if (!buildList) {
+                            buildList = "functional/OpenJcePlusTests,functional/security"
+                        }
+                    } else {
+                        VENDOR_TEST_REPOS = 'git@github.ibm.com:runtimes/test.git'
```

Comment:
These things don't need to be set when running the FIPS buildList?

Reply:
For FIPS, the decision is to run tests under `OpenJcePlusTests` and `security` folder only.